docs/conf/helpop-full.conf.example

```diff
@@ -927,11 +930,39 @@ using their cloak when they quit.">
                     channel (requires stripcolor module).
  T                  Blocks /NOTICEs to the channel from users who are
                     not at least halfop (requires nonotice module).
-
- g <mask>           Blocks messages matching the given glob mask
-                    (requires chanfilter module).
- X <mode>           Makes channel operators immune to the specified
-                    restrictive mode (requires exemptchanops module).
+ X <mode>:<status>  Makes users of <status> or higher immune to the
+                    specified restrictive mode. For example: flood:h
+                    (requires exemptchanops module).
+Possible restrictive modes for +X are:
+
+auditorium-see (requires the auditorium module)
+               Permission required to see the full user list of a +u channel
+auditorium-vis (requires the auditorium module)
+               Permission required to be visible in a +u channel
+blockcaps      (requires the blockcaps module)
+               Channel mode +B - Blocks messages with too many capital letters
```

Comment:
There's no need for a capital letter after a dash.

Reply:
Also, these descriptions describe what the mode does but the behaviour of the extban is generally the opposite of the mode (i.e. exempting matching users rather than punishing matching users).

Reply:
Maybe it's even better to just list the channel mode character they are, as the descriptions were just listed prior and we already say it's an exemption.